pyvrp/cpp/TimeWindowSegment.h

```diff
@@ -2,69 +2,75 @@
 #define TIMEWINDOWDATA_H
 
 #include "Matrix.h"
+#include "Measure.h"
 
 class TimeWindowSegment
 {
     using TWS = TimeWindowSegment;
 
-    int idxFirst = 0;  // Index of the first client in the segment
-    int idxLast = 0;   // Index of the last client in the segment
-    int duration = 0;  // Total duration, incl. waiting and servicing
-    int timeWarp = 0;  // Cumulative time warp
-    int twEarly = 0;   // Earliest visit moment of first client
-    int twLate = 0;    // Latest visit moment of last client
+    int idxFirst = 0;            // Index of the first client in the segment
+    int idxLast = 0;             // Index of the last client in the segment
+    duration_type duration = 0;  // Total duration, incl. waiting and servicing
+    duration_type timeWarp = 0;  // Cumulative time warp
+    duration_type twEarly = 0;   // Earliest visit moment of first client
+    duration_type twLate = 0;    // Latest visit moment of last client
 
-    [[nodiscard]] inline TWS merge(Matrix<int> const &durationMatrix,
+    [[nodiscard]] inline TWS merge(Matrix<duration_type> const &durationMatrix,
```

Comment:
I think the value of such an alias has lessened a bit, now that we have `Duration` (and similarly, `Distance`) rather than `duration_type`. `Matrix<Duration>` is already really expressive, I think. We should perhaps revisit this if we do use numpy arrays for these things. Those might not be as clear - I think pybind uses something like `py::array`.